app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt

```diff
@@ -34,6 +36,18 @@ class SuggestedEditsFeedClient(private var invokeSource: Constants.InvokeSource)
     override fun request(context: Context, wiki: WikiSite, age: Int, cb: FeedClient.Callback) {
         this.age = age
         cancel()
+
+        if (age == 0) {
+            // In the background, fetch the user's latest contribution stats, so that we can update whether the
+            // Suggested Edits feature is paused or disabled, the next time the feed is refreshed.
+            SuggestedEditsTasksFragment.updateStatsInBackground()
+        }
+
+        if (SuggestedEditsTasksFragment.isDisabled() || SuggestedEditsTasksFragment.maybePauseAndGetEndDate() != null) {
```

Comment:
I feel this knowledge of edit quality should be kept with a utility class available from all parts of the app, like AccountUtil class, rather than the Fragment being responsible for this. Or, maybe we can have the EditorTaskCounts class do all the calculations and offer isPausedForUser() and isDisabledForUser()? Not sure if Fragment is the best place for it. Come to think of it, a AccountUtil should probably calculate all stats info?

Reply:
Sure, good point. I went ahead and created a new utility class that holds this information.  I didn't put it into `EditorTaskCounts` since that's a model class, and neither into `AccountUtil` since I feel like that class shouldn't contain any state information other than user credentials.
lmk what you think.